### DISCUSSION TITLE
cn域名无法访问

### DISCUSSION THREAD

**Discussion by ShinoharaHaruna** [travellings-link/travellings#1308] [寻求帮助] [answered]
[travellings.cn](https://travellings.cn) 暂时无法访问.
网络环境: 济南教育网 & 境外节点
PING通, 但无法访问

**Comment by linlinzzo** [comment, marked as answer]
各位开往成员：
开往此前因为没有取得ICP备案，只能部署在海外服务器。开往维护组时常会收到开往被屏蔽、无法正常访问的反馈，我们也一直在尝试进行ICP备案。目前开往（travelling s.cn）的ICP备案材料已送交至福建省管局，根据腾讯云给出的消息，管局将在6到9天内审核。如果ICP备案审核通过，维护组将会逐步把开往迁移到国内，届时访问的速度和稳定性将得到极大的提升。
林林
2022年12月21日

**Comment by Bluemangoo** [comment]
只有 `www.travelings.cn` 是有用的。
你写的是`travelings.cn` 。
我刚刚也碰到这个问题。希望开往可以把`travelings.cn` 重定向到 `www.travelings.cn` ，或者反过来。

**Reply by linlinzzo** [reply]
> 只有 `www.travelings.cn` 是有用的。 你写的是`travelings.cn` 。 我刚刚也碰到这个问题。希望开往可以把`travelings.cn` 重定向到 `www.travelings.cn` ，或者反过来。

已设置显性URL转发。

**Reply by Bluemangoo** [reply]
> > 只有 `www.travelings.cn` 是有用的。 你写的是`travelings.cn` 。 我刚刚也碰到这个问题。希望开往可以把`travelings.cn` 重定向到 `www.travelings.cn` ，或者反过来。
> 
> 已设置显性URL转发。

其实302就够了。

**Reply by Bluemangoo** [reply]
> > > 只有 `www.travelings.cn` 是有用的。 你写的是`travelings.cn` 。 我刚刚也碰到这个问题。希望开往可以把`travelings.cn` 重定向到 `www.travelings.cn` ，或者反过来。
> > 
> > 已设置显性URL转发。
> 
> 其实302就够了。

问题好像在于显性转发。
